ios/Classes/SwiftAmplitudeFlutterPlugin.swift

```diff
@@ -35,7 +35,9 @@ import AmplitudeSwift
 
             amplitude?.logger?.debug(message: "Amplitude has been successfully initialized.")
 
-            // TODO(xinyi): add app lifecycle events
+            // Track DET app lifecycle: application installed and updated events
```

Comment:
Question: 
How can we disable this DET tracking?

Reply:
https://github.com/amplitude/Amplitude-Swift/blob/68171001367ae1b0856b930b6d2259113bf91fcd/Sources/Amplitude/Utilities/DefaultEventUtils.swift#L17